.doctrine-project.json

```
@@ -17,19 +17,27 @@
         "latest"
       ]
     },
+    {
+      "name": "3.2.x",
+      "branchName": "3.2.x",
+      "slug": "3.2.x",
+      "current": false,
+      "maintained": true
```

```suggestion
      "maintained": false
```

not sure about this one though.